src/video/wayland/SDL_waylandevents.c

```diff
@@ -355,6 +355,7 @@ pointer_handle_axis_common(struct SDL_WaylandInput *input,
     enum wl_pointer_axis a = axis;
     float x, y;
 
+    printf("axis common, value %d, axis %d\n", value, axis);
```

Comment:
Looks like some debug logging is still around here - we should remove prior to merging